packages/twenty-front/src/modules/object-record/record-field/meta-types/input/components/MultiItemFieldInput.tsx

```diff
@@ -110,6 +110,7 @@ export const MultiItemFieldInput = <T,>({
         item = items[index] as PhoneRecord;
         setInputValue(item.countryCode + item.number);
         break;
+      case FieldMetadataType.Array:
```

Comment:
Why is there no break condition for this? is the break condition for emails and arrays the same?

Reply:
Yes, the break statement for Array and Email are the same. The way the Array is being handled in the backend is just like the Email. So I found it better to use the same block as the Email one